www/posts/2020-05-25-v2-release-day.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: layouts/post.njk
+---
+
+After 40+ beta versions & release candidates we are so excited to introduce **Snowpack 2.0**, featuring:
+
+- A bundle-free dev environment that starts up in 50ms or less.
+- Application build tooling that stays fast as your codebase grows.
+- [Bundled production-optimized builds](/#snowpack-build) (powered by Webpack or Parcel)
+- [Hot Module Replacement (HMR)](/#hot-module-replacement) for React, Preact, Vue, Svelte & more.
+- Out-of-the-box support for TypeScript, JSX, CSS Modules & more.
+- [Custom build scripts](/#build-scripts) & [third-party plugins](/#build-plugins) to connect your favorite tools.
+
+For most Snowpack v1.x applications, all you need to do is run the following command and Snowpack 2.0 will walk you through updating any outdated configuration:
+
+```
+npm i snowpack@latest --save-dev
+```
+
+
+<br/>
+
+## Snowpack 2.0: Rethinking Best Practices
+
+!["Rethink established best practices" tweet.](/img/react-snarky-tweet-2.png)
+
+"Rethink established best practices." That was the famous antagonistic tweet directed at the React team back when React & JSX were first announced in 2013.
+
+It had been meant as a snarky burn against the project, but the React team turned it into a rallying cry. Pete Hunt even named [an entire JSConf talk](https://www.youtube.com/watch?v=DgVS-zXgMTk) after it, explaining why "rethinking best practices" was exactly what the web needed at that moment. 7 years later, and Pete was clearly proven right.
+
+**Snowpack is a chance to rethink best practices again, this time around web tooling.** The web has evolved over the last decade, but our build tooling hasn't kept up. The best practices of the previous decade no longer fit the requirements of today.
+
+**Snowpack is rethinking the biggest outdated assumption of all: the idea that web developers NEED bundlers.** Bundling continues to be great production optimization, dating all the way back to the early days of the web. It should continue to be available to those who want it. But... it was only in the last decade (with the rise of Browserify and later, Webpack) that bundling became a *required* part of your development workflow. 
```

Comment:
This one was clunky to process, lots of extra words (biggest, outdated, "of all", "the idea that"). Also was unclear if the statement following the `:` was the positive or negative assertion.

I would simplify the title, i.e.:
"Snowpack proves web developers don't NEED bundlers"

And follow up with the detail about standards, i.e.:
`We're challenging the outdated assumption that web devs need to use a bundler."

Reply:
Maybe restructure to "But bundling only became common in the last decade with the rise of Browserify and then Webpack"